Makefile

```diff
@@ -130,6 +130,15 @@ build-and-load-image: kind ## Build & load router/broker/controller image into t
 	$(CONTAINER_ENGINE) build $(CONTAINER_ENGINE_EXTRA_FLAGS) -t ghcr.io/kagenti/mcp-gateway:latest .
 	$(call load-image,ghcr.io/kagenti/mcp-gateway:latest)
 	kubectl rollout restart deployment/mcp-broker-router -n mcp-system 2>/dev/null || true
+	kubectl rollout restart deployment/mcp-controller -n mcp-system 2>/dev/null || true
+
+.PHONY: load-image
+load-image: kind ## Build & load router/broker/controller image into the Kind cluster	
+	$(call load-image,ghcr.io/kagenti/mcp-gateway:latest)	
+
+.PHONY: build-image
+build-image: kind ## Build & load router/broker/controller image into the Kind cluster	
+	$(CONTAINER_ENGINE) build $(CONTAINER_ENGINE_EXTRA_FLAGS) -t ghcr.io/kagenti/mcp-gateway:latest .
```

Comment:
The comments on these 2 new targets looks like a copy/paste.
Also, the 'rollout restart' is left out of the load-image section. Maybe that was intentional?
If there's value in these standalone targets, we can call them from `build-and-load-image` to avoid duplicate cmds